packages/SwingSet/test/upgrade/test-upgrade.js

```diff
@@ -133,6 +133,8 @@ const initKernelForTest = async (t, bundleData, config, options = {}) => {
   };
 };
 
+// Gratuitous change so I can create an otherwise identical PR
```

Comment:
`git commit -m 'Poke CI' --allow-empty` creates a no-changes commit that will pump CI.